frontend/src/components/dashboard/home/ActiveVisitorsCard.vue

```diff
@@ -106,20 +102,29 @@
         },
         created() {
             this.fetchingActivityDataItems();
+            this.fetchingActivityChartData();
```

Comment:
Please rename both methods to appropriate one, from the names of those methods it seems that you have recursive call